### PR TITLE
Added Telia LT TG389ac / GANT-1 17.1.7992 RBI

### PR DESCRIPTION
**Pull request by antnks**
With a help of my friend we were able to capture RBI of Telia Lithuania

It was located on an internal web server behind the VLAN5:
[http://10.0.98.250/TG389ac_gant-1_17.1.7992-0001012-20180515151413_PROD.rbi](http://10.0.98.250/TG389ac_gant-1_17.1.7992-0001012-20180515151413_PROD.rbi)

This firmware version needs CWMP mitm strategy (aka "tch-exploit" or "C#"), however it needs to be tuned to work with [http://10.0.61.42:7547/cpeserver/acs](http://10.0.61.42:7547/cpeserver/acs), as I couldn't make it working out of the box. Will release the code soon

### CODE REVIEW

File: docs/Repository.md [hunk 1]
```diff
@@ -96,6 +96,7 @@ A basic ADSL only BCM6362 based gateway. Very useful as SIP ATA.
...
 | 2      | 16.1.7053          | 2016-10-28 | #A            | [HTTPS](https://github.com/kevdagoat/hack-technicolor/raw/master/firmware/telia-gant-1_16.1.7053-1941003-bank_dump.xz) - *note: this **IS NOT** an RBI firmware, it is a raw bank dump, you can't use with TFTP or regular firmware upgrade tools*   |
 | 2      | 17.1.7992          | 2018-05-15 | #C            | [HTTPS](https://github.com/kevdagoat/hack-technicolor/raw/master/firmware/telia-gant-1_17.1.7992-0001012-bank_dump.xz) - *note: this **IS NOT** an RBI firmware, it is a raw bank dump, you can't use with TFTP or regular firmware upgrade tools*   |
+| 2      | 17.1.7992          | 2018-05-15 | #C            | [HTTPS](https://github.com/kevdagoat/hack-technicolor/raw/master/firmware/telia_gant-1_17.1.7992-0001012-20180515151413.rbi.xz) [HTTP VLAN5](http://10.0.98.250/TG389ac_gant-1_17.1.7992-0001012-20180515151413_PROD.rbi) |
```
Comment (LuKePicci) [inline docs/Repository.md:99]: Delete the rbi file form github, we don't host them here. Delete the bank dump too as we don't need it anymore.
Keep only the main original HTTP link from the ISP, remove the VLAN indication, use the * footnote instead as for others in this page (take the others form Telia as example).

Add the respective RBI torrent with that same HTTP link as webseed, upload the .torrent file here in this repo.

Comment (antnks) [reply docs/Repository.md:99]: Who is going to seed the torrent?

**Comment by LuKePicci** [comment]
Me at least
________________________________

File: docs/Repository.md (hunk 1 above)
Comment (antnks) [reply docs/Repository.md]: Done

**Comment by antnks** [comment]
Can you check now?

**Comment by LuKePicci** [comment]
You can also try grabbing the older ones by some [CPE emulation](https://github.com/LuKePicci/tr069-simulator). You can easily feed a compatible datamodel by converting clash get/getpn outputs

**Pull request merged**